test/tests.py

```diff
@@ -404,7 +404,6 @@ def test_0007_timeBetweenRequestsOnSameConnection(self):
         finally:
             proxy_single_connection_session.close()
 
-    @unittest.skip
     def test_0008_largeRequest(self):
```

Comment:
Since this test was flaky before, while in draft can you make the test repeat itself 10x and if we see they all pass, then we could remove the repeat logic and re-enable.  What do you think?

Reply:
Thanks, lets get it back in!

Reply:
https://github.com/opensearch-project/opensearch-migrations/actions/runs/9602550138/job/26483635785?pr=747

Reply:
Wasn't able to get it working with 99MIB, but was with 50 so moving to that
https://github.com/opensearch-project/opensearch-migrations/actions/runs/9602018665

Reply:
If we've got a change that runs with 10x iterations I'm ok merging that, with these recent iterations don't build as much confidence.